pkg/proc/linutil/regs_loong64_arch.go

```diff
@@ -0,0 +1,226 @@
+package linutil
+
+import (
+	"fmt"
+
+	"github.com/go-delve/delve/pkg/dwarf/op"
+	"github.com/go-delve/delve/pkg/dwarf/regnum"
+	"github.com/go-delve/delve/pkg/proc"
+)
+
+// LOONG64Registers is a wrapper for sys.PtraceRegs.
+type LOONG64Registers struct {
+	Regs       *LOONG64PtraceRegs // general-purpose registers
+	iscgo      bool
+	tp_tls     uint64
+	Fpregs     []proc.Register // Formatted floating point registers
+	Fpregset   []byte          // holding all floating point register values
+	loadFpRegs func(*LOONG64Registers) error
+}
+
+func NewLOONG64Registers(regs *LOONG64PtraceRegs, iscgo bool, tp_tls uint64,
+	loadFpRegs func(*LOONG64Registers) error) *LOONG64Registers {
+	return &LOONG64Registers{
+		Regs:       regs,
+		iscgo:      iscgo,
+		tp_tls:     tp_tls,
+		loadFpRegs: loadFpRegs,
+	}
+}
+
+// LOONG64PtraceRegs is the struct used by the linux kernel to return the GPR for
+// LoongArch64 CPUs. refer to sys/unix/zptrace_linux_loong64.go
```

Comment:
This structure refers to the definition of "PtraceRegs" in the sys/unix/ztype_linux_loong64.go file.